.github/workflows/integration-tests-api-tier-0-1.yml

```diff
@@ -0,0 +1,71 @@
+name: Integration tests for API - Tier 0 and 1
+
+on:
+  pull_request:
+    paths:
+        - ".github/workflows/integration-tests-api-tier-0-1.yml"
```

Comment:
When new changes are added inside `framework/wazuh/rbac/`, `test_rbac` should also be run.

Reply:
Done in new PR (due to duplicated commits in this branch)